src/demo/apps/DialerApp.tsx

```diff
@@ -77,6 +80,7 @@ export function DialerApp(props: AppProps): React.ReactElement {
 
     return (
         <div>
+            <ContextChannelSelector float={false} />
```

Comment:
Do you need to pass `float={false}` here if the default is already false?

Reply:
Cheers didn't notice that. Added defaultProps to the component to address this.